src/utils/config_utils.ts

```diff
@@ -22,4 +22,19 @@ export class ConfigUtils {
 		});
 		return repoPath || "";
 	}
+
+	getRepoIdByPath = (repoPath: string): number | null => {
+		if (!this.isConfigValid()) return null;
+		const repoBranchConfig = this.config.repos[repoPath]; 
+		if (!repoBranchConfig) return null;
+		return repoBranchConfig.id;
+	}
+
+	getFileIdByPath = (repoPath: string, branchName: string, fileName: string): number | null => {
```

Comment:
This function seems to have uncovered lines, kindly make sure we don't miss any case